README.md

```diff
@@ -36,7 +45,27 @@ FullStory.init({ orgId: '<your org id here>' });
 ReactDOM.render(<App />, document.getElementById('root'));
 ```
 
-## Examples
+Here's an example of initializing the SDK in an Angular app.
+
+```javascript
+import { Component } from '@angular/core';
+import * as FullStory from '@fullstory/browser';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+
+  constructor() {
+    FullStory.init({ orgId: '<your org id here>', recordCrossDomainIFrames: true });
```

Comment:
I love the addition of the Angular example! Wondering though if we should include `recordCrossDomainIFrames` option given that we kind of want people to think twice about doing this (per our security notice). This is more of an optics thing. Would it be a good idea to remove it from the example? It's fairly well documented.

Reply:
Switched to `debug` to play it safe.